src/content/app/genome-browser/components/browser-track-config/BrowserTrackConfig.tsx

```diff
@@ -15,163 +15,42 @@
  */
 
 import React from 'react';
-import { useDispatch, useSelector } from 'react-redux';
-import { OutgoingActionType } from '@ensembl/ensembl-genome-browser';
+import { useSelector } from 'react-redux';
+
+import RadioGroup, {
+  RadioOptions
+} from 'src/shared/components/radio-group/RadioGroup';
+import GeneTrackConfig from './track-config-views/GeneTrackConfig';
+import RegularTrackConfig from './track-config-views/RegularTrackConfig';
 
 import {
-  updateTrackConfigNames,
-  updateTrackConfigLabel,
-  updateApplyToAll,
-  updateApplyToAllTrackLabels,
-  updateApplyToAllTrackNames
+  getTrackType,
+  TrackType
 } from 'src/content/app/genome-browser/state/track-config/trackConfigSlice';
 
 import {
-  getTrackConfigNames,
-  getTrackConfigLabel,
   getApplyToAllConfig,
-  getBrowserCogTrackList,
   getBrowserSelectedCog
-} from 'src/content/app/genome-browser/state/track-config/trackConfigSelectors';
-
-import analyticsTracking from 'src/services/analytics-service';
+} from '../../state/track-config/trackConfigSelectors';
```

Comment:
Change the import path to `src/content/app/genome-browser/state/track-config/trackConfigSelectors`?